packages/features/ee/teams/components/MemberListItem.tsx

```diff
@@ -149,13 +149,11 @@ export default function MemberListItem(props: Props) {
               </Tooltip>
               {editMode && (
                 <Dropdown>
-                  <DropdownMenuTrigger
-                    asChild
-                    className="h-[36px] w-[36px] bg-transparent px-0 py-0 hover:bg-transparent focus:bg-transparent focus:outline-none focus:ring-0 focus:ring-offset-0">
```

Comment:
Most of this is handled by the Button component except the radix-state-open not present here; added to the Button component and removed all classNames.